<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string>Info</string>
  </property>
  <property name="statusTip">
   <string>Info</string>
  </property>
  <property name="whatsThis">
   <string>Info</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="Line" name="hline">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="illumiGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>440</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>Illuminating</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="illumiFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>400</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <widget class="QPushButton" name="illumiOkBtn">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>202</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>21</y>
       <width>271</width>
       <height>179</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="illumiVLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="illumiSpeeddLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="illumiSpeedLabel">
          <property name="text">
           <string>Speed (km/h)</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="illumiSpeedSpinBox">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="illumiLevelLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="illumiLevelLabel">
          <property name="text">
           <string>ILL Level</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="illumiLevelSpinBox">
          <property name="maximum">
           <number>63</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="illumiStateLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="illumiStateLabel">
          <property name="text">
           <string>ILL State</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="illumiStatesLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="illumiStateOff">
            <property name="text">
             <string>OFF</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="illumiStateOn">
            <property name="text">
             <string>ON</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="illumiStateNone">
            <property name="text">
             <string>NONE</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="illumiDriveLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Drive</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="illumiDrivesLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QCheckBox" name="illumiPkb">
            <property name="text">
             <string>PKB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="illumiIg">
            <property name="text">
             <string>IG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="illumiRev">
            <property name="text">
             <string>REV</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="illumiBrakeLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="illumiBrakeLabel">
          <property name="text">
           <string>Brake</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="illumiBrakesLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QCheckBox" name="illumiBrakeTest">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="illumiBrakePark">
            <property name="text">
             <string>Park</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="illumiBrakeP">
            <property name="text">
             <string>P</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="illumiBrakeN">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="revGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>440</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>REV</string>
   </property>
   <widget class="QFrame" name="revFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>14</y>
       <width>162</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="revVLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="revPkbLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="revRev">
          <property name="text">
           <string>REV</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="revPkb">
          <property name="text">
           <string>PKB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="revSpeedLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="revSpdLabel">
          <property name="text">
           <string>Speed (km/h)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="revSpdSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="revOkBtn">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>59</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
